### ISSUE TITLE
Fix migrate user to new site button

### ISSUE THREAD

**Issue by ebarry** [publiclab/plots2#424]
### What happened just before the problem occurred

As Admin user, i clicked the button on an old profile to migrate to new site and got an error message: 

![screen shot 2016-03-16 at 5 01 35 pm](https://cloud.githubusercontent.com/assets/161439/13828663/020c70ce-eb99-11e5-97e6-ee23a580c864.png)

https://publiclab.org/admin/migrate/20

![screen shot 2016-03-16 at 5 03 43 pm](https://cloud.githubusercontent.com/assets/161439/13828673/13376a5c-eb99-11e5-8eb2-97f9931ea7b7.png)

### Relevant URLs

https://publiclab.org/profile/lpercifield

### PublicLab.org username

@liz (@ebarry on here)

### Browser, version, and operating system

Chrome OSX 10.11.3

**Comment by jywarren** [comment]
I see in the logs:

````
Started GET "/admin/migrate/20" for 173.225.73.97 at 2016-03-16 21:07:38 +0000
Processing by AdminController#migrate as HTML
  Parameters: {"id"=>"20"}
Started GET "/profile/gabirielle" for 68.180.230.48 at 2016-03-16 21:07:38 +0000
Processing by UsersController#profile as */*
  Parameters: {"id"=>"gabirielle"}
  Rendered notes/_notes.html.erb (1.1ms)
Completed 500 Internal Server Error in 902.3ms

ActiveRecord::RecordNotUnique (Mysql2::Error: Duplicate entry '20' for key 'PRIMARY': INSERT INTO `rusers` (`created_at`, `crypted_password`, `current_login_at`, `current_login_ip`, `email`, `failed_login_count`, `id`, `last_login_at`, `last_login_ip`, `last_request_at`, `login_count`, `openid_identifier`, `password_salt`, `persistence_token`, `photo_content_type`, `photo_file_name`, `photo_file_size`, `reset_key`, `role`, `updated_at`, `username`) VALUES ('2016-03-16 21:07:38', NULL, NULL, NULL, 'lpercifield@gmail.com', 0, 20, NULL, NULL, NULL, 0, NULL, NULL, 9104889, NULL, NULL, NULL, NULL, 'basic', '2016-03-16 21:07:38', 'lpercifield')):
  lib/authlogic_openid/authlogic_openid/acts_as_authentic.rb:74:in `save'
  app/models/drupal_users.rb:162:in `migrate'
  app/controllers/admin_controller.rb:193:in `migrate'
````

I believe this is because of #227, where we haven't fully synced `DrupalUser.uid` with `User.id`. #227 is a bit complex, but if there cease to be uid/id collisions, the existing code will be able to create the new record for lpercifield.

**Comment by jywarren** [comment]
I believe this is fixed as we just resolved #227
